db/schema.rb

```diff
@@ -188,17 +188,17 @@
 
   create_table "question_group_conditionals", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
     t.integer  "rapidfire_question_group_id"
-    t.integer  "cohort_id"
+    t.integer  "campaign_id"
     t.datetime "created_at",                  null: false
     t.datetime "updated_at",                  null: false
-    t.index ["cohort_id"], name: "index_question_group_conditionals_on_cohort_id", using: :btree
+    t.index ["campaign_id"], name: "index_question_group_conditionals_on_campaign_id", using: :btree
```

Comment:
Is this regenerated from migrations, or did you find-and-replace the schema.rb file?

I have no idea how indexes interact with renaming and mgirations.